database/duties_repository.js

```diff
@@ -0,0 +1,48 @@
+import { client } from '../connections.js';
+
+export const dbName = 'CallOfDuty';
+export const dutiesDBCollection = 'Duties';
+
+async function addNewDuty(newDuty) {
+  const dutyToInsert = {
+    name: newDuty.name,
+    location: newDuty.location,
+    time: newDuty.time,
+    constraints: newDuty.constraints,
+    soldiersRequired: newDuty.soldiersRequired,
+    value: newDuty.value,
+  };
+  dutyToInsert.soldiers = [];
```

Comment:
```suggestion
   const dutyToInsert = {
    name: newDuty.name,
    location: newDuty.location,
    time: newDuty.time,
    constraints: newDuty.constraints,
    soldiersRequired: newDuty.soldiersRequired,
    value: newDuty.value,
    soldiers: [];
  };
```

I think you did the same in the soldier collection... you can fix it in this pr

Reply:
fixed